examples/ownable/Cargo.toml

```diff
@@ -7,14 +7,27 @@ publish = false
 version = "0.0.0"
 
 [dependencies]
-openzeppelin-stylus = { path = "../../contracts", features = ["ownable", "erc20"] }
+openzeppelin-stylus = { path = "../../contracts", features = [
+  "ownable",
+  "erc20",
+] }
 alloy-primitives.workspace = true
 stylus-sdk.workspace = true
 stylus-proc.workspace = true
 mini-alloc.workspace = true
 
+[dev-dependencies]
+alloy.workspace = true
+e2e = { path = "../../lib/e2e" }
+tokio = { version = "1.12.0", features = ["full"] }
+eyre = "0.6.8"
```

Comment:
I don't have a strong reason, but it's the same as for `tokio` -- we don't use eyre in the contracts, only in the tests and in `e2e`.

I have to form an opinion, will revisit tomorrow.

Reply:
Sure thing!

Reply:
Yeah, this is better, because this way we make sure that every crate is using the same version of a package.

I'll create an issue to do this not only for eyre, but for everything else as well.

Reply:
https://github.com/OpenZeppelin/rust-contracts-stylus/issues/144